src/OAuth2.php

```diff
@@ -694,6 +703,7 @@ public function buildFullAuthorizationUri(array $config = [])
             'redirect_uri' => $this->redirectUri,
             'state' => $this->state,
             'scope' => $this->getScope(),
+            'enable_serial_consent' => $this->partConsent,
```

Comment:
Just do the logic here that you have below. If this is a boolean (as it should be), it becomes very easy:
```php
'enable_serial_consent' => $this->partialConsent ? 'true' : 'false',
```